xml/operations-troubleshooting-ts_blockstorage.xml

```diff
@@ -315,36 +315,13 @@ ansible-playbook -i hosts/verb_hosts _cinder_post_check.yml</screen>
    larger than the Virtual Size (raw size) of your image or creation will fail
    with an error.
   </para>
-  <para>
-   An error such as this would appear in <literal>cinder-volume.log</literal>
-   file:
-  </para>
-<screen>'2016-06-14 07:44:00.954 25834 ERROR oslo_messaging.rpc.dispatcher ImageCopyFailure: Failed to copy image to volume: qemu-img: /dev/disk/by-path/ip-192.168.92.5:3260-iscsi-iqn.2003-10.com.lefthandnetworks:mg-ses:146:volume-c0e75c66-a20a-4368-b797-d70afedb45cc-lun-0: error while converting raw: Device is too small
-2016-06-14 07:44:00.954 25834 ERROR oslo_messaging.rpc.dispatcher'</screen>
-  <para>
-   In an example where creating a 1GB bootable volume fails, your image may
-   look like this:
-  </para>
-<screen>$ qemu-img info /tmp/image.qcow2
-image: /tmp/image.qcow2
-file format: qcow2
-virtual size: 1.5G (1563295744 bytes)
-disk size: 354M
-cluster_size: 65536
-...</screen>
-  <para>
-   In this case, note that the image format is qcow2 and hte virtual size is
-   1.5GB, which is greater than the size of the bootable volume. Even though
-   the compressed image size is less than 1GB, this bootable volume creation
-   will fail.
-  </para>
   <para>
    When creating your disk model for nodes that will have the cinder volume
-   role make sure that there is sufficient disk space allocated for a temporary
+   role, make sure that there is sufficient disk space allocated for temporary
    space for image conversion if you will be creating bootable volumes. You
-   should allocate enough space to the filesystem as would be needed to cater
-   for the raw size of images to be used for bootable volumes - for example
-   Windows images can be quite large in raw format.
+   should allocate enough space to the filesystem as would be needed to contain
```

Comment:
Remove "you should" and just have: "Allocate enough space to the filesystem to contain the raw..."